### PR TITLE
Fix motion primitives CI fail

### PR DESCRIPTION
**Pull request by mkhansen-intel**
---

## Basic Info
Attempting to fix the build farm CI in #473

### CODE REVIEW

**Comment by mkhansen-intel** [comment]
@SteveMacenski - Will this fix the latest build issue?

File: nav2_motion_primitives/include/nav2_motion_primitives/motion_primitive.hpp [hunk 1]
```diff
@@ -79,9 +79,7 @@ class MotionPrimitive
     RCLCPP_INFO(node_->get_logger(), "%s attempting behavior", taskName_.c_str());
 
     ResultMsg result;
-    nav2_tasks::TaskStatus status;
-
-    status = onRun(command);
+    nav2_tasks::TaskStatus status = onRun(command);
```
Comment (orduno) [inline nav2_motion_primitives/include/nav2_motion_primitives/motion_primitive.hpp]: Looks ok, but also could be
```cpp
auto status = onRun(command);
```

**Comment by crdelsey** [comment]
I was able to reproduce the problem by doing a release build.
```
colcon build --symlink-install --cmake-args -DCMAKE_BUILD_TYPE=Release
```

Unfortunately this change didn't fix the problem.

**Comment by orduno** [comment]
> I was able to reproduce the problem by doing a release build.
> 
> ```
> colcon build --symlink-install --cmake-args -DCMAKE_BUILD_TYPE=Release
> ```
> Unfortunately this change didn't fix the problem.

Line 102 should be
```cpp
auto status = nav2_tasks::TaskStatus::FAILED;
```

**Comment by orduno** [comment]
I've pushed the fix to the PR.

**Comment by crdelsey** [comment]
That fixed it.

**Pull request merged**